spring-cloud-stream-schema/pom.xml

```diff
@@ -47,4 +49,27 @@
 			<scope>test</scope>
 		</dependency>
 	</dependencies>
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.avro</groupId>
+				<artifactId>avro-maven-plugin</artifactId>
+				<version>${avro.version}</version>
+				<executions>
+					<execution>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>schema</goal>
+						</goals>
+
+					</execution>
+				</executions>
+				<configuration>
+					<testOutputDirectory>${project.basedir}/target/generated-test-sources</testOutputDirectory>
+					<sourceDirectory>${project.basedir}/src/main/resources/</sourceDirectory>
```

Comment:
I removed surce directory, but we need the test configuration otherwise it will fail